src/test/java/org/sagebionetworks/bridge/spring/controllers/ParticipantControllerTest.java

```diff
@@ -1287,77 +1262,6 @@ public void getActivityHistoryV3DefaultsToNulls() throws Exception {
         assertNull(endTimeCaptor.getValue());
     }
 
-    @Test
-    public void updateIdentifiersWithPhone() throws Exception {
```

Comment:
Was this API deleted from ParticipantController? Did we forget to include ParticipantController in the PR?

Reply:
I've restored all these tests. At one point I thought this could be moved entirely to the Accounts controller, but I'm not 100% sure this is never used by clients to add email/phone, or never would be, so I've left it.